ingest_find_existing_asset.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  ingest_find_existing_asset_name = "${local.environment}-dr2-ingest-find-existing-asset"
+}
+
+module "ingest_find_existing_asset" {
+  source          = "git::https://github.com/nationalarchives/da-terraform-modules//lambda"
+  function_name   = local.ingest_find_existing_asset_name
+  handler         = "uk.gov.nationalarchives.ingestfindexistingasset.Lambda::handleRequest"
+  timeout_seconds = 60
+  policies = {
+    "${local.ingest_find_existing_asset_name}-policy" = templatefile(
+      "./templates/iam_policy/ingest_check_preservica_for_existing_io_policy.json.tpl", {
```

Comment:
1. This still has the old name
2. And the actual `json.tpl` file still has the old nam

Reply:
I've renamed the policy file which has updated the templatefile call.